src/main/java/com/nasrpi/filesharing/UploadFileResponse.java

```diff
@@ -0,0 +1,57 @@
+package com.nasrpi.filesharing;
+
+/**
+ * Exception class for FileNotFoundException
+ * 
+ * @author grandolf49
+ */
+public class UploadFileResponse {
+
+	private String fileName;
+
+	private String fileDownloadUri;
+
+	private String fileType;
+
+	private long size;
+
+	public UploadFileResponse(String fileName, String fileDownloadUri, String fileType, long size) {
+		this.setFileName(fileName);
+		this.setFileDownloadUri(fileDownloadUri);
+		this.setFileType(fileType);
+		this.setSize(size);
+	}
+
+	public String getFileName() {
```

Comment:
use lombok for all getters and setters in this class

Reply:
done